### ISSUE TITLE
[HLS] manifest with chapters - playback problem

### ISSUE THREAD

**Issue by mtr81** [peak3d/inputstream.adaptive#1285]
I get from server HLS manifests:
[manifest_1.txt](https://github.com/xbmc/inputstream.adaptive/files/11710685/manifest_1.txt)
[manifest_2.txt](https://github.com/xbmc/inputstream.adaptive/files/11710686/manifest_2.txt)
It's live streaming, but if I use ISA the playback starts from first TS file (not current time). This problem doesn't exist during playback via inputstream.ffmpegdirect.
After adding properties:
```
play_item.setProperty('ResumeTime', '43200')
play_item.setProperty('TotalTime', '1')
```
playback started from correct point, but after 10 minutes player back to first segment.

**Comment by L2501** [comment]
old issue
https://github.com/xbmc/inputstream.adaptive/issues/574
https://github.com/xbmc/inputstream.adaptive/issues/1066

**Comment by mtr81** [comment]
Not exactly. In this case problem is incorrect #EXT-X-DISCONTINUITY support.

**Comment by L2501** [comment]
it's still the same issue regardless of stream. Kodi doesn't keep track of the chapters on manifest updates.
Solution is also the same: use proxy to remove the chapters.

more info also on this issue: https://github.com/xbmc/inputstream.adaptive/issues/584

**Comment by CastagnaIT** [comment]
I haven't tested but i'm pretty sure it's the same problem i solved long time ago on DASH
https://github.com/xbmc/inputstream.adaptive/blob/6b8cb3707b985f74446ad9e2781053a30f7870c6/src/parser/DASHTree.cpp#L138-L157
I think it's never been fixed on HLS because a decent solution is not so simple to do like on dash,
in order to fix this other things would have to be revised and changed in some way as well
